pynapple/process/decoding.py

```diff
@@ -38,6 +48,12 @@ def decode_1d(tuning_curves, group, ep, bin_size, time_units="s", feature=None):
     feature : Tsd, optional
         The 1d feature used to compute the tuning curves. Used to correct for occupancy.
         If feature is not passed, the occupancy is uniform.
+    smoothing_windowsize : int
+        Size of the Gaussian window to be used for smoothing the data. Default value is None.
+    smoothing_std : float
+        Standard deviation of the Gaussian kernel for smoothing. Default value is 0.
+    norm : bool
```

Comment:
You should explain more here what it does. When computing a spike count, it smooth it and preserves the spike count in the 0 bin.